### PR TITLE
creates LetterBtn component

### PR DESCRIPTION
**Pull request by pozniej-znajde-wolne-haslo**

### CODE REVIEW

File: src/components/LetterGrid/LetterGrid.tsx [hunk 1]
```diff
@@ -24,17 +24,13 @@ export const LetterGrid = ({ gameLetters, setWordInput }: LetterGridProps) => {
...
-            <button
-              className={classNames(styles.letter, styles.flex)}
-              value={letter}
-              onClick={handleUserInput}
-            >
-              {letter}
-            </button>
-          </div>
-        ))}
+        {gameLetters.map((letter) =>
```
Comment (druckmax) [inline src/components/LetterGrid/LetterGrid.tsx:27]: You can make this shorter:

```js
const isPrimary = letter === gameLetters[0]

<LetterBtn variant={isPrimary ? "primary" : "secondary"} key={letter} letter={letter} onClick={handleUserInput} />
```

File: src/components/LetterBtn/LetterBtn.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import { MouseEvent } from 'react';
+import classNames from 'classnames';
+import styles from './letterBtn.module.scss';
+
+type LetterBtnProps = {
+  variant: string;
```
Comment (druckmax) [inline src/components/LetterBtn/LetterBtn.tsx:6]: To be more concise you can do `variant: "primary" | "secondary"`.
This way typescript does not allow any other values than those two.

File: src/components/LetterBtn/letterBtn.module.scss [hunk 3]
```diff
@@ -0,0 +1,22 @@
+@import '../../styles/colors.scss';
```
Comment (druckmax) [inline src/components/LetterBtn/letterBtn.module.scss:1]: We can use an import alias for this:

```css
@import '@/styles/colors.scss';
```

File: src/components/LetterGrid/LetterGrid.tsx (hunk 1 above)
Comment (pozniej-znajde-wolne-haslo) [reply src/components/LetterGrid/LetterGrid.tsx:27]: where should the const isPrimary be declared?

File: src/components/LetterBtn/letterBtn.module.scss (hunk 3 above)
Comment (pozniej-znajde-wolne-haslo) [reply src/components/LetterBtn/letterBtn.module.scss:1]: ach cool. didn't know that. corrected it in all scss files (u used absolute path in the footer too, I just copy you :D

**Pull request merged**